source/docs/software/advanced-controls/introduction/approaches-to-ctrl-sys-design.rst

```diff
@@ -0,0 +1,77 @@
+
+They are not mutually exclusive - many techniques will use these concepts simultaneously.
+
+Start with FeedForward
+-----------------------
+
+Because most FRC mechanisms can be manufactured to be have well-defined behavior and can be easily modeled, it's generally encouraged to look into feed-forward techniques first. 
+
+How do you expect your system to behave?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step in designing a control system is writing down information on how you expect your mechanism to behave.
+
+This step is done by combining one or more concepts you may be familiar with from Physics: drawing free body diagrams of the forces that act on the mechanism, taking measurements of mass and moment of inertia from your CAD models, applying standard models of how DC motors or pneumatic cylinders convert energy into mechanical force and motion, etc.
+
+Models do not have to be extremely accurate to be useful. For example, a very simple model of a motor is that its speed is proportional to the voltage applied. A motor which spins at 3000 RPM at 12v will spin near 1500RPM at 6V. If you know you want your motor to be spinning at 1254 RPM, you can use the voltage/speed ratio to calculate the amount of voltage to send to the motor.
+
+`ReCalc is an online calculator <https://www.reca.lc/>`__ which helps take care of some of these physics model calcualtions for common FRC mechanisms.
+
+
+System Identification
+^^^^^^^^^^^^^^^^^^^^^
+
+A key way to improve the accuracy of a simple physics model is to perform experiments on the real mechanism, record data, and use the data to *derive* the constants associated with different parts of the model.
+
+This is useful if CAD models are not perfectly accurate, or parts of the model are not easily predicted (ex: friction in a gearbox).
+
+:doc:`FRC's system identification tool </docs/software/pathplanning/system-identification/introduction>` supports some common FRC mechanisms, including drivetrains. It can be configured to load custom code onto a robot, exercise the mechanism and record data, and derive relevant constants to build a more accurate model of the system behavior.
+
+
+Combining Feedback and FeedForward
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Even with lots of experiments and study, it is not possible to know every force that will be exerted on a robot's mechanism with great detail. For example, in a flywheel shooter, the timing and exact forces associated with a ball being put through the mechanism are not easy to define accurately. For another example, consider the fact that gearboxes can be greased to reduce friction, but throw off grease over time and get more friction. This is a complex process to model well.
+
+In cases where not everything can be known, incorporating sensors and *feedback* control is useful to account for inaccuracies in the model of system behavior.

```

Comment:
Be a bit more forceful than "this is useful for."  Feedback is *necessary* to account for inaccuracies in the system model - it just might not be necessary to do that in the context of any given control task.